### PR TITLE
explaining formulas and params

### PR DESCRIPTION
**Pull request by liuchunchi**

### CODE REVIEW

File: draft-bclp-green-terminology.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ helps to identify the “hot spots” and key components in a system with regard
 
 ## Energy efficiency Metrics and Measurement Method
 
-Metric for DSLAM, MSAN,GPON, GEPON equipment:
+Metric for DSLAM, MSAN, GPON, GEPON equipment:
```
Comment (boucadair) [inline draft-bclp-green-terminology.md:149]:
```suggestion
Metrics for DSLAM, MSAN, GPON, and GEPON equipment:
```

File: draft-bclp-green-terminology.md [hunk 2]
```diff
@@ -146,7 +146,7 @@ helps to identify the “hot spots” and key components in a system with regard
 
 ## Energy efficiency Metrics and Measurement Method
```
Comment (boucadair) [inline draft-bclp-green-terminology.md:147]:
```suggestion
## Energy Efficiency Metrics and Measurement Methods
```

File: draft-bclp-green-terminology.md (hunk 1 above)
Comment (boucadair) [reply draft-bclp-green-terminology.md:149]: Consider adding a reference

File: draft-bclp-green-terminology.md [hunk 3]
```diff
@@ -173,17 +173,27 @@ The proposed metric for router and Ethernet switches is
...
+                        Ti = a*Tu1 + b*Tu2 + c*Tu3
+                        Pw = a*Pu1 + b*Pu2 + c*Pu3
+
+where (a, b, c) is the relative weight at different usage percentage, a+b+c=1
+
+(Pu1, Pu2, Pu3) is the power at different usage percentage
+
+(Tu1, Tu2, Tu3) is the throughput at different usage percentage
+
 Metric for small network devices:
```
Comment (boucadair) [inline draft-bclp-green-terminology.md:185]:
```suggestion
Metrics for small network devices:
```

File: draft-bclp-green-terminology.md [hunk 4]
```diff
@@ -173,17 +173,27 @@ The proposed metric for router and Ethernet switches is
...
 
 The metrics adopted for small networking devices intended for home/domestic or small office use is
```
Comment (boucadair) [inline draft-bclp-green-terminology.md:187]: "adopted": consider adding a reference or clarify this was adopted by whom.

File: draft-bclp-green-terminology.md [hunk 5]
```diff
@@ -173,17 +173,27 @@ The proposed metric for router and Ethernet switches is
...
+                        EER = (0.35T_idle+0.5T_lowpower+0.15T_Maximum)/(0.35P_idle+0.5P_lowpower+0.15P_maximum) [Mbit/s/W]
 
-Where throughput is maximum non drop data rate beween wide area Network and
-local area network kport in the ingress direction;
+Where throughput is maximum non-drop data rate beween Wide Area Network and
+Local Area Network ingress port;
 Line rate/speed is maximum possible number of transmitted/received bits.
 Power shall be averaged over 5 minutes, taking measurements every 30 seconds.
-During idle power,IP pings shall be sent via the user interface.
+During idle power, IP pings shall be sent via the user interface.
```
Comment (boucadair) [inline draft-bclp-green-terminology.md:195]:
```suggestion
During idle power, IP pings shall be sent via the user interface.
```

Why this is mentioned here?

File: draft-bclp-green-terminology.md [hunk 6]
```diff
@@ -173,17 +173,27 @@ The proposed metric for router and Ethernet switches is
...
+0.35, 0.5 and 0.15 are the example relative weights defined in {{L.1310}}.
```
Comment (boucadair) [inline draft-bclp-green-terminology.md:196]:
```suggestion
0.35, 0.5, and 0.15 are relative weights examples as defined in {{L.1310}}.
```

**Review by boucadair — CHANGES_REQUESTED** [review]

File: draft-bclp-green-terminology.md (hunk 5 above)
Comment (billwuqin) [reply draft-bclp-green-terminology.md:212]: I also suggest to remove this sentence since it is not mandatory to have this context information.

Comment (liuchunchi) [reply draft-bclp-green-terminology.md]: removed for out of context

File: draft-bclp-green-terminology.md (hunk 4 above)
Comment (liuchunchi) [reply draft-bclp-green-terminology.md:204]: these definitions are from {{L.1310}}, added references.

File: draft-bclp-green-terminology.md [hunk 7]
```diff
@@ -188,19 +188,28 @@ The proposed metric for router and Ethernet switches is
...
+Metrics for small network devices:
 
 The metrics adopted for small networking devices intended for home/domestic or small office use is
 
- EER = (0.35Tidle+0.5Tlowpower+0.15TMaximum)/(0.35Pidle+0.5Plowpower_0.15Pmaximum) (Mbit/s/W)
+                        EER = (0.35T_idle+0.5T_lowpower+0.15T_Maximum)/(0.35P_idle+0.5P_lowpower+0.15P_maximum) [Mbit/s/W]
 
-Where throughput is maximum non drop data rate beween wide area Network and
-local area network kport in the ingress direction;
+Where throughput is maximum non-drop data rate beween Wide Area Network and
```
Comment (boucadair) [inline draft-bclp-green-terminology.md:208]:
```suggestion
Where throughput is maximum non-drop data rate between ingress/egress ports.
```

Comment (boucadair) [reply draft-bclp-green-terminology.md:208]: WAN/LAN taxonomy does not apply for core devices.

File: draft-bclp-green-terminology.md [hunk 8]
```diff
@@ -188,19 +188,28 @@ The proposed metric for router and Ethernet switches is
...
+Local Area Network ingress port;
```
Comment (boucadair) [inline draft-bclp-green-terminology.md:209]:
```suggestion
```

File: draft-bclp-green-terminology.md (hunk 4 above)
Comment (billwuqin) [reply draft-bclp-green-terminology.md:204]: works for me.

**Review by boucadair — CHANGES_REQUESTED** [review]

**Review by billwuqin — APPROVED** [review]

**Pull request merged**